milestone2.ts

```diff
@@ -125,7 +131,9 @@ let kyle: Volunteer = {
 /* Lets make a clone of Kyle above using the spread operator and assign it to kyleClone
  */
 type CopyVolunteer = (vol: Volunteer) => Volunteer;
-let copyVolunteer: CopyVolunteer; // Code here
+let copyVolunteer = (vol: Volunteer): Volunteer => {
```

Comment:
Didn't use the provided type here, not wrong of course but it's more work for you!